cornell-dti/course-plan
# Added new profile container with the first page of settings

### Summary 

Created a new profile container which shows up when Tools are enabled and when you click on the profile button. This new profile container so far only contains the first page to edit the profile, the rest will be added in a future PR. This is the first step towards implementing the profile redesign that Kehui made. 



Remaining TODOs:
- Add the rest of the sections required to edit the profile and the buttons to navigate them
- The templates will also go into this container in the future. 
- Add the confirmation modal when submitting changes

### Test Plan 

- Make sure that when tools are disabled, the page has the default behaviour of opening the onboarding modals
- Make sure that when editing the profile, the changes are only saved when clicking submit and that no changes are saved when exiting prematurely
- Make sure that switching between the navbar pages functions correctly
